legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/DDL/testDDL.pure

```diff
@@ -112,4 +112,45 @@ function <<test.Test>>  meta::relational::tests::ddl::testSetupDataSqlGeneration
    'insert into personTable (ID,FIRSTNAME,LASTNAME,AGE,ADDRESSID,FIRMID,MANAGERID) values (1,\'Peter\',\'Smith\',23,1,1,2);'
    ];
   assertSameElements($sqls,$expectedSqls);
-}  
\ No newline at end of file
+}  
+
+function <<test.Test>>  meta::relational::tests::ddl::testSetupDataSqlGenerationWithColumnValueHasDelimiterAndQuotes():Boolean[1]
+{
+  let records ='default\n'+
+                'personTable\n'+
+                'id, firstName, lastName, age, addressId, firmId, managerId\n'+
+                '1,Peter,  "I\'m Smith, Jr",23,1,1,2';
```

Comment:
I think the issue is the following test case:
`'1,Peter's,  "I\'m Smith, Jr"  ,23,1,1,2'`
We need to add logic to handle the case when quote is not the first character

Reply:
I don think `'1,Peter's,  "I\'m Smith, Jr"  ,23,1,1,2' ` is right. 
for this, it should be `'1, "Peter\'s",  "I\'m Smith, Jr"  ,23,1,1,2'`